### PR TITLE
News and events fix

### LINKED ISSUE #43: Fix `more news` and `more events` url in home page
These urls are defined in the ELIXIR toolkit theme and should get an optional path parameter

### PR DESCRIPTION
**Pull request by bedroesb**
This will close #43.

### CODE REVIEW

**Comment by bedroesb** [comment]
+ fixing missing issues

**Comment by rabuono** [comment]
The image [googledoc_way_flow.svg](https://github.com/elixir-europe/infectious-diseases-toolkit/pull/59/files#diff-c94a7c799a439e7d7441f56e259d91d273825225d3539ffca35561e0ac4cf390) has "Contributing to the **RDMkit** website - Google doc way". Might be best to remove `RDMkit` and keep just "Contributing - Google doc way".
Can we change and use that image?

File: index.html [hunk 1]
```diff
@@ -87,10 +87,10 @@ <h2>We welcome contributors!</h2>
             <div class="container">
                 <div class="row">
                     <div class="col-lg-6">
-                    {% include news.html caption=true title=true limit=3 %}
+                    {% include news.html caption=true title=true limit=3 caption_url="/about/news" %}
                     </div>
                     <div class="col-lg-6 mt-3 mt-lg-0">
-                    {% include events.html caption=true title=true event_type="upcoming_event" %}
+                    {% include events.html caption=true title=true event_type="upcoming_event" caption_url="/about/events" %}
```
Comment (rabuono) [inline index.html:93]: Does this remove the "upcoming event" portion of the landing page? Might be best to not have it there as it would be empty.

**Review by rabuono — CHANGES_REQUESTED** [review]
Just the small comment on the image and the question on the events
Thanks!

File: index.html (hunk 1 above)
Comment (bedroesb) [reply index.html:93]: yes, since there are no upcoming events, the whole block will not be shown ;)

**Review by rabuono — APPROVED** [review]

**Pull request merged**